src/components/hero.js

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+import { Box, Text, Button } from "rebass"
+import { css } from "@emotion/core"
+import { heroImg, heroBrandmark } from "../images"
+import Container from "./container"
+
+const heroStyle = css`
+    position: relative;
+    left: 50%;
+    right: 50%;
+    background: url("${heroImg}") no-repeat center;
+    width: 100vw;
+    height: 80rem;
+    margin-left: -50vw;
+    margin-right: -50vw;
```

Comment:
I would suggest removing this wrapper box `<Box m="0 auto" maxWidth={960} px={1}>` in layout.js then remove `left`, `right`, `margin-left` and `margin-right` properties. My main concern for that wrapping jsx is it limits the flexibility of the layout and having us resort to negative margins to achieve full width components. We could just stack containers if needed.